test/test_suite.py

```diff
@@ -723,7 +723,7 @@ def _test_get_table_names(self, schema=None, table_type="table", order_by=None):
 
             if order_by == "foreign_key":
                 answer = ["users", "user_tmp", "email_addresses", "dingalings"]
-                eq_(table_names, answer)
+                eq_(set(table_names), set(answer))
```

Comment:
Use set to compare as test is flaky due to order of list's element.

Reply:
IN other PR's workflow the related tests were failing(assertion) due to table has already exist, delete the table manually and try to run the tests plenty of times, its running fine.

Reply:
You can write `answer = {"users", "user_tmp", "email_addresses", "dingalings"}` one line upper to create a set instead of creating a list and then converting it.

Reply:
Multiple ways, your suggestion will save one conversion.